integration-tests/docker-sws-api/elasticsearch/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM elasticsearch:5.6.16
+COPY ./scripts /usr/share/elasticsearch/config/scripts
+COPY ./synonym.txt /usr/share/elasticsearch/config/synonym.txt
```

Comment:
Instead of adding a Dockerfile, you can just map these files directly into the container at runtime via the `docker-compose.yml`.  Check out [the config section of the web container on digital platform](https://github.com/NCIOCPL/cgov-digital-platform/blob/develop/docker/docker-compose.yml#L14-L17) for an example.

The advantage of doing it this way is you don't have to remember to rebuild the image when the files change.